www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/location.xml

```diff
@@ -61,6 +61,12 @@
     <style>selectpicker</style>
     <help>Select an upstream to proxy to or connect via FastCGI if chosen.</help>
   </field>
+  <field>
+    <id>location.new_urlpattern</id>
+    <label>New location path</label>
+    <type>text</type>
+    <help>Select a new path for upstream to proxy.</help>
```

Comment:
this is a textbox, you cannot select. `Enter` or `Define` would be a lot better. Also explain what is doing.